requirements.txt

```diff
@@ -3,7 +3,7 @@ chainer>=4.0.0
 fastcache; python_version<'3.2'
 funcsigs; python_version<'3.5'
 future
-gym>=0.9.7,<=0.12.1
+gym>=0.9.7
```

Comment:
Note:
Looks good because the new method names are implemented in openai/gym#836 (0.9.6).